projects/igniteui-angular/src/lib/grids/grid/grid.component.html

```diff
@@ -229,6 +230,9 @@
         </ng-template>
     </div>
     <div class="igx-grid__scroll-end" [style.float]='"right"' [style.width.px]='pinnedWidth' [style.min-width.px]='pinnedWidth' [hidden]="pinnedWidth === 0 || isPinningToStart"></div>
+    <div class="igx-grid__grid-addrow-snackbar">
+        <igx-snackbar #addRowSnackbar [actionText]="snackbarActionText" (onAction)="navigateToAddedRow($event)" displayTime='2000'>{{snackbarLabel}}</igx-snackbar>
```

Comment:
Since this will always show the "Show" action even if the added row is visible, perhaps we can conditionally hide the action if the added row is in view.